packages/sanity/src/structure/comments/src/components/reactions/CommentReactionsBar.tsx

```diff
@@ -150,38 +147,39 @@ export const CommentReactionsBar = React.memo(function CommentReactionsBar(
               reactionName={name}
               userIds={userIds}
             >
-              <ReactionButtonCard
-                __unstable_focusRing
-                border
-                disabled={readOnly}
-                forwardedAs="button"
-                // eslint-disable-next-line react/jsx-no-bind
-                onClick={() => handleSelect(name)}
-                padding={2}
-                radius={6}
-                tone={hasReacted ? 'primary' : 'transparent'}
-                type="button"
-              >
-                <Flex align="center" gap={2}>
-                  <Text size={1}>{emoji}</Text>
-
-                  <Text size={1} weight={hasReacted ? 'medium' : undefined}>
-                    {reactionsList?.length}
-                  </Text>
-                </Flex>
-              </ReactionButtonCard>
+              <TransparentCard tone="default">
+                <UIButton
+                  disabled={readOnly}
+                  mode="ghost"
+                  // eslint-disable-next-line react/jsx-no-bind
+                  onClick={() => handleSelect(name)}
+                  padding={2}
+                  radius="full"
```

Comment:
`radius="full"` is a recent addition to Sanity UI, we don't really use it that often, but this is a good case for it